etc/build.py

```diff
@@ -145,6 +148,10 @@ def package_scripts(build_root, config_only=False, windows=False):
                                               os.path.join(build_root, CANNED_DIR[1:]),
                                               os.path.join(build_root, CANNED_DIR[1:], "*json")),
             shell=True, print_output=True)
+        run("cp {} {} && chmod 644 {}".format(PROTOBOARDS_SCRIPTS,
```

Comment:
Why not use python instead of exec?

Reply:
just mimicking how we were doing it for canned scripts, can change but should change both in that case.

Reply:
Then LGTM